packages/dx-react-scheduler/src/plugins/day-view.tsx

```diff
@@ -108,22 +125,28 @@ class DayViewBase extends React.PureComponent<VerticalViewProps> {
           }}
         />
 
+        <Getter
+          name="timeCellsData"
+          computed={this.timeCellsDataComputed(viewName, startDayHour, endDayHour)}
+        />
+
```

Comment:
Can we move this getter in BasicView? I know it needs only in Week and Day. Maybe conditional getter rendering or conditional getter computing will be helpful